upup/pkg/fi/cloudup/populate_cluster_spec.go

```diff
@@ -289,6 +289,7 @@ func (c *populateClusterSpec) run() error {
 		switch m {
 		case "config":
 			// Note: DefaultOptionsBuilder comes first
+			codeModels = append(codeModels, &components.EtcdOptionsBuilder{Context: optionsContext})
```

Comment:
Let's put this after  DefaultOptionsBuilder, just so we don't have to change the comment :-)